ui/apps/mapping/src/components/ScanDataDialog/ScanDataDialog.tsx

```diff
@@ -38,14 +38,37 @@ const EMPTY_DBCONNECTION_FORM_DATA = {
   schema: "",
 };
 
+const DELIMITERS = [
+  {
+    name: ",",
+    value: ",",
+  },
+  {
+    name: ";",
+    value: ";",
+  },
+  {
+    name: "|",
+    value: "|",
+  },
+  {
+    name: "Tab",
```

Comment:
[white rabbit](https://github.com/data2evidence/d2e-WhiteRabbit/blob/master/whiterabbit/src/main/java/org/ohdsi/whiteRabbit/WhiteRabbitMain.java#L124) accepts `tab` as input instead of `\t`